src/state-incentive-details.tsx

```diff
@@ -187,28 +182,37 @@ const LinkButton: FC<PropsWithChildren<{ href: string }>> = ({
   </a>
 );
 
-const renderIncentiveCard = (key: Key, incentive: Incentive) => (
```

Comment:
I turned this into its own component because passing the Key around felt like not what you're supposed to do